app/main/views/g_cloud.py

```diff
@@ -596,6 +596,14 @@ def did_you_award_contract(framework_family, project_id):
     form = DidYouAwardAContractForm()
     if form.validate_on_submit():
         if form.did_you_award_a_contract.data == form.STILL_ASSESSING:
+            data_api_client.mark_direct_award_project_as_still_assessing(
+                project_id=project_id,
+                user_email=current_user.email_address)
+            flash(
+                Markup(f"Your response for ‘{project['name']}’ has been saved.<br/>"
```

Comment:
This looks like an injection vulnerability to me - if you're wrapping in `Markup` then you're claiming the content is safe. But you haven'y escaped project name?

I don't think you need the `Markup` at all - a plain string would be fine?

Reply:
Oh, except you have a `<br>`... I suppose you will need to escape the project name separately then.

Reply:
Have removed the `<br>` as it feels wrong to have a line break here style-wise.